src/hardware/TX/Jumper T-Lite.json

```diff
@@ -17,7 +17,7 @@
     "power_max": 3,
     "power_default": 2,
     "power_control": 0,
-    "power_values": [-10,-6,-3,6],
+    "power_values": [-11,-7,-4,1],
```

Comment:
Freshly measured my T-Lite v2 w. a RF meter:

10mW => 12 mW
25mW => 28 mW
50mW => 54 mW
100mW => 105 mW

In my case, -12 works closer (=10.7mW) than -11 in the 10 mW case. But it's possibly within the margin of error.